modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm

```diff
@@ -567,6 +567,11 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	item_path = /obj/item/clothing/head/imperial/white
 	restricted_roles = list(JOB_CAPTAIN, JOB_HEAD_OF_PERSONNEL, JOB_BLUESHIELD, JOB_HEAD_OF_SECURITY, JOB_RESEARCH_DIRECTOR, JOB_QUARTERMASTER, JOB_CHIEF_MEDICAL_OFFICER, JOB_CHIEF_ENGINEER)
 
+/datum/loadout_item/head/nun
```

Comment:
the icon files aren't modular either

Reply:
So you are saying you are requesting the new art to be placed in a new individual dmi? Located where, if so? Please give more detail.

Reply:
You would have to look around for other chaplain items that we have added on Skyrat, and probably add those sprites in that same .dmi file. I can't remember off the top of my head if we do (I'm pretty sure we do though?), and in case we don't, then you'll have to add them to one of the modules.